charts/passbolt/templates/deployment.yaml

```diff
@@ -56,6 +56,22 @@ spec:
             - name: DATASOURCES_DEFAULT_PASSWORD
               value: {{ .Values.mariadb.auth.password }}
             {{- end }}
+            - name: EMAIL_TRANSPORT_DEFAULT_HOST
+              value: "{{ .Values.email.host }}"
+            - name: EMAIL_TRANSPORT_DEFAULT_PORT
+              value: {{ .Values.email.port }}
```

Comment:
Be careful : you must always use a string for environment values. 

You may replace this section with :
```
- name: EMAIL_TRANSPORT_DEFAULT_PORT
  value: "{{ .Values.email.port }}"
```